DataFormats/simulation/include/SimulationDataFormat/MCInteractionRecord.h

```diff
@@ -0,0 +1,40 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @brief  Simulated interaction record
+
+#ifndef ALICEO2_MCINTERACTIONRECORD_H
+#define ALICEO2_MCINTERACTIONRECORD_H
+
+#include <Rtypes.h>
+#include <iostream>
+
+namespace o2
+{
+struct MCInteractionRecord {
+  double timeNS = 0.; ///< time in NANOSECONDS from start of run (period=0, orbit=0)
```

Comment:
We were actually discussing this today in WP1. Shouldn't this be defined there, rather than by simulation alone? For example you use an int for bc, but if I understand correctly that will be only 12 bits in raw data. @mkrzewic

Reply:
Hi
This is a timestamp for mc only, providing the collision (rather than lhc clock) time in ns, like fairrootana does. I just completed this with bc, orbit period in case we will need to implement mc trigger later.

Reply:
@ktf My answer was incomplete: actually I've defined the limits for bc/orbit/period counters in https://github.com/shahor02/AliceO2/blob/9f556051ace2acbd2e363a6204aa8124e2b76eb4/Steer/include/Steer/InteractionSampler.h, but eventually they should be moved to some global constants header, as the comment in PR says.